bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java

```diff
@@ -89,7 +99,27 @@ public EnsemblePlacementPolicy initialize(ClientConfiguration conf,
      * @throws BKNotEnoughBookiesException if not enough bookies available.
      * @return the java.util.ArrayList<org.apache.bookkeeper.net.BookieSocketAddress>
      */
-    public ArrayList<BookieSocketAddress> newEnsemble(int ensembleSize, int writeQuorumSize, int ackQuorumSize, Map<String, byte[]> customMetadata, Set<BookieSocketAddress> excludeBookies) throws BKNotEnoughBookiesException;
+    public default ArrayList<BookieSocketAddress> newEnsemble(int ensembleSize, int writeQuorumSize,
+        int ackQuorumSize, Map<String, byte[]> customMetadata, Set<BookieSocketAddress> excludeBookies) throws BKNotEnoughBookiesException {
+        return newEnsemble(ensembleSize, ackQuorumSize, excludeBookies);
+    }
+
+    /**
+     * Legacy method introduce for compatibility with 4.4 clients
+     * @param ensembleSize
+     *          Ensemble Size
+     * @param quorumSize
+     *          Quorum Size
+     * @param excludeBookies
+     * @return
+     * @throws org.apache.bookkeeper.client.BKException.BKNotEnoughBookiesException
+     *
+     * @see #newEnsemble(int, int, int, java.util.Map, java.util.Set)
+     */
+    public default ArrayList<BookieSocketAddress> newEnsemble(int ensembleSize, int quorumSize,
```

Comment:
as the prev comment, BookKeeper needs to call "new methods" which in turn will call "old methods".
In my case I always have subclasses of DefaultEnsemblePlacementPolicy, so I can change the public interface as you are proposing at make only subclasses of DefaultEnsemblePlacementPolicy call the 'legacy' methods